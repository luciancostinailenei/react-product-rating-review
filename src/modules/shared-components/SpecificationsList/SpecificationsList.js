import React from 'react'

const SpecificationsList = () => {
  return (
    <ul className="list list--no-padding-left list--squared">
      <li>Lorem ipsum dolor sit amet, consectetur </li>
      <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit</li>
      <li>Lorem ipsum dolor sit amet, consectetur adipiscing </li>
      <li>Lorem ipsum dolor</li>
      <li>Lorem ipsum dolor sit amet</li>
    </ul>
  )
}

export default SpecificationsList
