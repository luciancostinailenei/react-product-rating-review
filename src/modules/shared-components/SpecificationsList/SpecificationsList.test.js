import React from 'react'
import toJson, { createSerializer } from 'enzyme-to-json'
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

import './SpecificationsList'
import SpecificationsList from './SpecificationsList'

configure({ adapter: new Adapter() })

describe('ProductReview', () => {
  it('matches snapshot', () => {
    const SpecificationsListWrapper = shallow(<SpecificationsList />)

    expect(toJson(SpecificationsListWrapper)).toMatchSnapshot()
  })
})
