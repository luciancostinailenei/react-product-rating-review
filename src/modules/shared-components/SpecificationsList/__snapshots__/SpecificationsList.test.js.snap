// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductReview matches snapshot 1`] = `
<ul
  className="list list--no-padding-left list--squared"
>
  <li>
    Lorem ipsum dolor sit amet, consectetur 
  </li>
  <li>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit
  </li>
  <li>
    Lorem ipsum dolor sit amet, consectetur adipiscing 
  </li>
  <li>
    Lorem ipsum dolor
  </li>
  <li>
    Lorem ipsum dolor sit amet
  </li>
</ul>
`;
